### ISSUE TITLE
Errors when tapping on SVG

### ISSUE THREAD

**Issue by PMPepper** [ftlabs/fastclick#387]
The following was all tested on android 4.2.2. I'm not certain if this issue occurs on other devices/browsers.

I have some links and buttons containing SVG graphics (as elements, not as background images). Sometimes, when tapping on these, I get the following error on line 553 ("targetTagName = targetElement.tagName.toLowerCase();"):

Uncaught TypeError: Cannot call method 'toLowerCase' of undefined

Basically, the node that has received the touch event is of type SVGElementInstance, and this does not appear to have the tagName property. In addition, I'm getting a similar error from line 217 ("switch (target.nodeName.toLowerCase()) {") which again appears to be caused by SVGElementInstance not having the property 'nodeName'

**Comment by newjam** [comment]
I'm seeing this too.

**Comment by yinzSE** [comment]
```stylus
svg
svg *
  pointer-events: none
```
@newjam it didn't work with me.
